src/com/lftechnology/java/training/niraj/shuffle/ListShuffle.java

```diff
@@ -0,0 +1,120 @@
+
+package com.lftechnology.java.training.niraj.shuffle;
+
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Creates an array list with 49 Integer objects containing the numbers 1
+ * through 49. It then randomly shuffles the list and selects the first 6 values
+ * from the shuffled list. Finally, it sorts the selected values and prints them
+ * out.
+ * 
+ * @author Niraj Rajbhandari
+ */
+public class ListShuffle {
+
+	private static final Logger LOGGER =
+		Logger.getLogger(ListShuffle.class.getName());
+	private UserNumInputImpl userInput;
+
+	private ListShuffle() {
+
+		userInput = new UserNumInputImpl();
+	}
+
+	public static void main(String[] args) {
+
+		try (Scanner scanner = new Scanner(System.in)) {
+			ListShuffle listShuffle = new ListShuffle();
+			int subListLength = listShuffle.getSubListLength(scanner);
+			List<Integer> numberList =
+				listShuffle.getNumberList(scanner, subListLength);
+			List<Integer> sortedList =
+				listShuffle.getSortedSubList(numberList, subListLength);
+			LOGGER.log(Level.INFO, "Sorted List : {0}", sortedList);
+		}
+		catch (InputMismatchException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)